exercise_01.py

```diff
@@ -0,0 +1,18 @@
+def displayPattern(n):
+    n_to_print = ""
+    total = ""
+    for i in range(1, n + 1):
+        n_to_print = "{} {}".format(i, n_to_print)
+        space = "  " * (n - i)
+
+        total =  "{}\n{}{}".format(total, space, n_to_print)
+
+    return total
+
+input = eval(input("Enter a number: "))
```

Comment:
It's great you added the `main` function, and the `if __name__ == '__main__':` construct, but you should not have zero-indentation level calls as this one. Apply the same in the other programs. This means that this kind of calls should be done in your main function.